apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -124,6 +133,37 @@ const aichatSlice = createSlice({
         chatMessage.status = status;
       }
     },
+    setLevelAiCustomizations: (
+      state,
+      action: PayloadAction<LevelAiCustomizations>
+    ) => {
+      state.levelAiCustomizations = action.payload;
+    },
+    setLevelAiCustomizationProperty: (
```

Comment:
These reducers heavily borrowed from here:

https://github.com/code-dot-org/code-dot-org/blob/18a5815741461ba0c6984129db5eee68a5275829/apps/src/lab2/levelEditors/aiCustomizations/EditAiCustomizations.tsx#L69-L99

Feel a little torn on the best way to manage all of these...`levelAiCustomizations` has some top level properties that have simple values (temperature, systemPrompt, etc), some that have a complicated value (modelCardInfo), and one that doesn't have a visibility (`hidePresentationPanel`). So, sort of a mixed bag.